test/extended/util/annotate/generated/zz_generated.annotations.go

```diff
@@ -847,6 +847,8 @@ var annotations = map[string]string{
 
 	"[Top Level] [sig-arch] Managed cluster should have operators on the cluster version": "have operators on the cluster version [Suite:openshift/conformance/parallel]",
 
+	"[Top Level] [sig-arch] Managed cluster should only include cluster daemonsets that have maxUnavailable update of 10 percent": "should only include cluster daemonsets that have maxUnavailable update of 10 percent [Suite:openshift/conformance/parallel]",
```

Comment:
[verify][1]:

```
FAILURE after 0.032s: hack/verify-generated.sh:14: executing 'git diff --quiet /go/src/github.com/openshift/origin/test/extended/util/annotate/generated/' expecting success: the command returned the wrong error code
There was no output from the command.
There was no error output from the command.
```

Not very helpful.  Hopefully a fluke.  But [GCP][2]:

```
panic: unable to find test [Top Level] [sig-arch] Managed cluster should only include cluster daemonsets that have maxUnavailable update of 10 or 33 percent
```

seems more serious.  I guess that's this line needing a regen, although [I'm also fine dropping the 33% back out][3].

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/25928/pull-ci-openshift-origin-master-verify/1367128290422165504#1:build-log.txt%3A46
[2]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/25928/pull-ci-openshift-origin-master-e2e-gcp/1367128286907338752#1:build-log.txt%3A98
[3]: https://github.com/openshift/origin/pull/25928#discussion_r586725460